dev-tools/cosmos-records/delete-orders.ts

```diff
@@ -13,6 +13,4 @@ export default function deleteOrders() {
   deleteDocuments('consolidations', 'consolidationId', 'SELECT * FROM c');
 }
 
-if (require.main === module) {
-  deleteOrders();
-}
+deleteOrders();
```

Comment:
What was this about?  Why were we doing this check to begin with?

Reply:
I added that guard a couple tickets ago. I broke `npm run reload-orders` with it. So this reverts it.